tests/sentry/api/endpoints/test_assistant.py

```diff
@@ -38,8 +42,7 @@ def test_activity(self):
         })
         assert resp.status_code == 201
         resp = self.client.get(self.path)
-        assert resp.status_code == 200
-        assert resp.data == {k: v for k, v in GUIDES.items() if v['id'] != 2}
+        assert resp.status_code == 201
```

Comment:
Huh? why is the get returning 201 instead of 200?

Reply:
fixed